apsuite/commissioning_scripts/measure_beta.py

```diff
@@ -188,7 +188,7 @@ def anly_method(self, value):
 
     def start(self):
         """."""
-        if self._thread.is_alive():
+        if not self.is_online or self._thread.is_alive():
```

Comment:
@murilobalves, I am using the name `isonline` istead of `is_online` in the coupling class. I did this because of the `ismeasuring` property. Which one do you prefer?

Reply:
Ok, let's keep this pattern